docs/sources/setup-grafana/configure-security/configure-authentication/saml-ui/index.md

```diff
@@ -0,0 +1,44 @@
+---
+description: Learn how to configure SAML authentication in Grafana's UI.
+menuTitle: Configure SAML authentication in UI
+title: Configure SAML authentication in Grafana's UI
+weight: 1150
+---
+
+# Configure SAML authentication in Grafana
+
+> **Note:** Available in [Grafana Enterprise]({{< relref "../../../introduction/grafana-enterprise/" >}}) version 10.0 and later, and [Grafana Cloud Pro and Advanced](/docs/grafana-cloud/).
+
+You can configure SAML authentication in Grafana through the user interface (UI) or the Grafana configuration file. For instructions on how to set up SAML using the Grafana configuration file, refer to [Configure SAML authentication in Grafana]({{< relref "../saml/" >}}).
+
+The Grafana SAML UI provides the following advantages over configuring SAML in the Grafana configuration file:
+
+- It is accessible by hosted Grafana users
+- It doesn't require Grafana to be restarted after a configuration update
+- Access to the SAML UI only requires access to authentication settings, so it can be used by users with limited access to Grafana's configuration
+
+> **Note:** Configuration in the UI takes precedence over the configuration in the Grafana configuration file. SAML settings from the UI will override any SAML configuration set in the Grafana configuration file.
+
+# Before you begin
+
+To follow this guide, you need:
+
+- Knowledge of SAML authentication. Refer to [SAML authentication in Grafana]({{< relref "../saml/" >}}) for an overview of Grafana's SAML integration.
+- Permissions that allow you to read and update authentication settings. These permissions are granted by `fixed:authentication.config:writer` role.
+  By default, this role is granted to Grafana server administrator in self-hosted instances and to Organization admins in hosted Grafana instances.
+
+# Configure SAML
```

Comment:
```suggestion
## Steps
```

Reply:
I'm remember what we should do here. for topics that have just a single task (so one set of steps with one outcome), we use Steps here and then you add a simple intro like this: "To configure SAML using the Grafana UI, complete the following steps:"

And that's it. no need for any of the other navigation oriented content that you have. That stuff will be make its way into the steps.